io/private/EptSupport.hpp

```diff
@@ -90,6 +158,127 @@ inline std::array<double, 3> toArray3(const Json::Value& json)
     return result;
 }
 
+class PDAL_DLL Key
```

Comment:
Does this need PDAL_DLL?

Reply:
Don't know, these DLL exports were already here - presumably they were needed.